Added info about Radix CLI command scale, options previous and imageT… (#204)
equinor/radix-public-site: public-site/docs/src/docs/topic-radix-cli/index.md

rx logout
```

> Radix CLI uses Radix API to execute operations. An option `verbose` can be used to get more details about requests and responses to and from the Radix CLI:

Reviewer: ```suggestion
Radix CLI uses Radix API to execute operations. Use the option `verbose` to get more details about requests and responses to and from the Radix CLI:
```

Reviewer: resolved in private discussion, keep "as is"

    ```
* Create a new "deploy only" pipeline job
    ```shell
    rx create job deploy --application your-app-name --environment dev
    rx create job deploy -a your-app-name -e dev
    ```

> * Create a new "deploy only" pipeline job with specified image tags. When `radixconfig.yaml` contains `image` option with dynamic [imageTagName](https://radix.equinor.com/references/reference-radix-config/#imagetagname), this `imageTagName` can be altered in the Radix CLI `create job deploy` command argument. This argument will override value, defined in the `radixconfig.yaml`. This argument can be defined for multiple components in the command. `imageTagName`, provided as argument if the command `rx create job deploy` is shown in the Radix pipeline orchestration job log. If a component name, specified in this argument, does not exist in the Radix application environment - such argument is ignored

Reviewer: ```suggestion
* Create a new "deploy only" pipeline job with specified image tags. When `radixconfig.yaml` contains `image` option with dynamic [imageTagName](https://radix.equinor.com/references/reference-radix-config/#imagetagname), this `imageTagName` can be altered in the Radix CLI `create job deploy` command argument. This argument will override values defined in the `radixconfig.yaml`. This argument can be defined for multiple components in the command. When `imageTagName` is provided as an argument in the command `rx create job deploy` it will be shown in the Radix pipeline orchestration job log. Component names that does not exist within the Radix application environment will be ignored.
```

> * Create a new "deploy only" pipeline job with specified image tags. When `radixconfig.yaml` contains `image` option with dynamic [imageTagName](https://radix.equinor.com/references/reference-radix-config/#imagetagname), this `imageTagName` can be altered in the Radix CLI `create job deploy` command option `image-tag-name`. This option will override value, defined in the `radixconfig.yaml`, it can be defined for multiple components in the command. `image-tag-name`, provided as an option in the command `rx create job deploy` is shown in the Radix pipeline orchestration job log. If a component name, specified in this option, does not exist in the Radix application environment - such option is ignored

Reviewer: ```suggestion
* Create a new "deploy only" pipeline job with specified image tags. When `radixconfig.yaml` contains `image` option with dynamic [imageTagName](https://radix.equinor.com/references/reference-radix-config/#imagetagname), this `imageTagName` can be altered in the Radix CLI `create job deploy` command option `image-tag-name`. This option will override values defined in the `radixconfig.yaml` and can be defined for multiple components in the command. `image-tag-name`, provided as an option in the command `rx create job deploy` is shown in the Radix pipeline orchestration job log. Component names that does not exist within the Radix application environment will be ignored.
```

    ```shell
    rx get logs environment -a your-app-name --environment qa
    ```

> * Get previous (terminated) container log of a Radix application component. This helps to figure out why the container was restarted. Such log is not always available as Kubernetes node, whe the pod with this container was running, can be removed or restarted.  

Reviewer: ```suggestion
* Get previous (terminated) container log of a Radix application component. This may help to indicate why the container was restarted. These logs are not always available as the Kubernetes node, where the pod with this container was running on, may have been removed or restarted.  
```

    rx start application -a your-app-name
    rx restart application -a your-app-name
    ```

> * Scale up or down Radix application component replicas. Allowed values between "0" and "20" (value "0" is an equivalent of the command `rx stop`). Scaling can be useful for tuning of resource configuration to figure out how amount of replicas affect performance of an application and particular need of CPU and memory. After re-deployment, start or restart, scaled component gets replicas, specified in the `radixconfig.yaml`, "1" if not specified, or set by [horizontal scaling](https://radix.equinor.com/references/reference-radix-config/#environmentconfig)  

Reviewer: ```suggestion
* Scale up or down Radix application component replicas. Allowed values between "0" and "20" (value "0" is an equivalent of the command `rx stop`). Scaling can be useful for tuning the resource configuration to figure out what amount of replicas affect performance of an application and particular need of CPU and memory. After re-deployment, start or restart, scaled component gets replicas, specified in the `radixconfig.yaml`, "1" if not specified, or set by [horizontal scaling](https://radix.equinor.com/references/reference-radix-config/#environmentconfig)  
```
